src/operator/LimitsField.tsx

```diff
@@ -18,13 +18,17 @@ interface Props {
 }
 
 const LimitsField: FC<Props> = ({type, name, limits, onChangeLimits}) => {
+  const [hasFocus, setHasFocus] = useState(false)
   const {hasWritePermissions} = useContext(OperatorContext)
   const value = get(limits, name, '')
+  const csv = isNaN(value)
```

Comment:
Nit: you might consider changing this from `csv` - I thought it was an actual .csv file. And technically, it'll only be comma separated in some regions.

One thing that stands out is that we're doing two different comparisons. This code checks against `isNaN` while the code below checks the type. Is it possible to make these type checks consistent?

Reply:
Good call on both. I changed this place to check the `InputType`.